ci/unit_tests/workspace_addjson/main_data/workspace_forAddJsonTest.json

```diff
@@ -0,0 +1,4785 @@
+{
```

Comment:
I think you should not push this file as it is a by-product of testing.

Reply:
gone